treestats_paper.tex

```diff
@@ -685,18 +685,19 @@ \subsection*{Branch statistics}
 where $\ell_k(c,d)$ is the length of the region in $[c,d)$ that the tree $T_k$ covers
 (i.e., if $T$ covers the half-open interval $[a_{k-1},a_k)$,
 then $\ell_k(c,d) = \max(0, \min(d,a_k) - \max(c,a_{k-1}))$).
-The \emph{polarised} version of a Branch statistic
+The \emph{polarized} version of a Branch statistic
 is defined analogously to the Node and Site versions:
 \begin{align} \label{eqn:branch_polarised}
-    \text{(polarised)} \qquad
+    \text{(polarized)} \qquad
     \branchp(f, \iw)_T
     &=
     \sum_{u \in T} \beta_T(u) f(\nw_{T}(u)) ,
 \end{align}
 and $\branchp(f, \iw)_{[c,d)}$ is defined using $\branchp(f, \iw)_T$ as before.
-However, the correspondence between the polarised statistics is less tight
-than for the unpolarised ones.
+However, the correspondence between the polarized statistics is less tight
+than for the unpolished ones.
```

Comment:
"unpolished"

Reply:
Fixed; but see my comment below: what does this mean?

Reply:
Hm: I think I may have been confused when I wrote that: duality should hold under infinite-sites for polarized stats also. We should strike this sentence.